tests/integration/targets/ntnx_karbon_clusters_node_pools/tasks/create_node_pools.yml

```diff
@@ -0,0 +1,177 @@
+---
+- name: Generate random node_pool name 
+  set_fact:
+    random_name: "{{query('community.general.random_string',numbers=false, special=false,length=9,upper = false)[0]}}"
+
+- set_fact:
+    suffix_name: "ansible"
+
+- set_fact:
+    node1_name:  "{{random_name}}{{suffix_name}}1"
+    node2_name:  "{{random_name}}{{suffix_name}}2"
+    node3_name:  "{{random_name}}{{suffix_name}}3"
+
+- debug:
+    msg: Start testing ntnx_karbon_clusters_node_pools
+
+- name: Create node pool  with subnet uuid 
```

Comment:
How can we create node pools without creating a k8 cluster? This test would be dependent on k8s cluster as test setup

Can we add this test in CRUD tests of karbon clusters ? Since we are creating a k8s cluster there we can club this test cases post update tests of karbon clusters.

Reply:
Assuming k8s cluster present would lead to maintaining proper setup every time we run karbon tests